### DISCUSSION TITLE
Updating while in the process of screening (with ASReview on DRE)

### DISCUSSION THREAD

**Discussion by EsGroot** [asreview/asreview#1235] [Q&A] [answered]
We work on a review with ASReview installed at the DRE (Digital Research Environment), which is used for collaboration with people outside the UMCU. In the meantime, we saw the YouTube movie on exporting ris files from ASReview. I have two questions:

(1) from which version of ASReview is this ris export possible and where do I have to search for the version that is installed on the DRE environment? 

(2) if the ris export is not possible with the version of ASREview that we have now and we need to do an update (which is a bit less straightforward in DRE than 'normal'), are there any specifics that we need to be careful about, so not run the risk of loosing our screening that we did thus far as the result of an update? 

Thanks for your recommendations, best wishes, Esther

**Comment by J535D165** [comment, marked as answer]
I hope the following answers your questions:

> (1) from which version of ASReview is this ris export possible and where do I have to search for the version that is installed on the DRE environment?

https://github.com/asreview/asreview/releases/tag/v0.19

- Version 1.0 -> bottom of the settings menu
- Version 0.x -> top left menu (under the title)

Please keep in mind that RIS export, requires RIS import. 

> (2) if the ris export is not possible with the version of ASREview that we have now and we need to do an update (which is a bit less straightforward in DRE than 'normal'), are there any specifics that we need to be careful about, so not run the risk of loosing our screening that we did thus far as the result of an update?

It is worth upgrading to version 1. Version 1 has a better interface, is more stable, and has lots of new features. Don't worry about upgrading your projects, they will work in v1 as well (after conversion).

Just a general note, a backup of your projects is always advised (also when you are not upgrading). Simply copy the contents of your `~/.asreview` folder.
